copis/gui/panels/properties.py

```diff
@@ -125,7 +125,7 @@ def on_points_selected(self, points: List[int]) -> None:
         """On core_p_selected, set to point view."""
 
         if len(points) == 1:
-            action = self.c.actions[points[0] - len(self.c.devices)]
+            action = self.parent.core.actions[points[0] - len(self.parent.core.devices)]
```

Comment:
Just use `self.core` here

Reply:
We tried that yesterday and every broke, remember? something about each subpanel being it's own entity?